tests/unit/utils/test_gmm.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import numpy as np
+from nequip.utils import gmm
+from sklearn import mixture
+
+fit_data = np.random.rand(10, 3)
+gmm_sklearn = mixture.GaussianMixture(
+    n_components=3, covariance_type="full", random_state=123
+)
+
+gmm_torch = gmm.GaussianMixture(n_components=3, seed=123, n_features=3)
+# gmm_torch = torch.jit.script(gmm_torch)
+
+
+class TestGMM:
+    def test_fit_forward_simple(self):
+        gmm_sklearn.fit(fit_data)
+        gmm_torch.fit(torch.from_numpy(fit_data))
+
+        assert torch.allclose(
+            torch.from_numpy(gmm_sklearn.means_), gmm_torch.state_dict()["means"]
+        )
+
+        assert torch.allclose(
+            torch.from_numpy(gmm_sklearn.covariances_),
+            gmm_torch.state_dict()["covariances"],
+        )
+
+        assert torch.allclose(
+            torch.from_numpy(gmm_sklearn.weights_), gmm_torch.state_dict()["weights"]
+        )
+
+        assert torch.allclose(
+            torch.from_numpy(gmm_sklearn.precisions_cholesky_),
+            gmm_torch.state_dict()["precisions_cholesky"],
+        )
+
+        test_data = np.random.rand(20, 3)
```

Comment:
Always use some fixed seed you make up for unit tests; otherwise it is extremely difficult to reproduce failures which can be very confusing. (Can also lead to nondeterminism with regard to test ordering, which is bad; for example on GitHub actions we run our tests in two threads, so they run in a different order than when you run them in one thread locally)